app-libs/exchange-oracle/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "ita-exchange-oracle"
+version = "0.8.0"
+authors = ["Integritee AG <hello@integritee.network>"]
+edition = "2018"
+
+[features]
+default = ["std"]
+std = [
+    "thiserror",
+    "url",
+    "itc-rest-client/std",
+    "serde/std",
+    "serde_json/std",
+    "log/std",
```

Comment:
I like these to be ordered alphabetically (the same goes for the dependencies further below). Otherwise it's quite tedious to get a good overview